irods_specific_implementation/constants.hpp

```diff
@@ -5,7 +5,8 @@
 
 namespace irods::rest
 {
-    const std::string base_url = "/irods-rest/1.0.0";
+    // IRODS_CLIENT_VERSION is a macro that is defined by the CMakeLists.txt file.
+    const std::string base_url = IRODS_CLIENT_VERSION;
```

Comment:
i think we still need the leading `/irods-rest/`?

Reply:
good catch